src/Helper/GitHelper.php

```diff
@@ -727,11 +727,7 @@ public function getGitDir()
             return $this->topDir;
         }
 
-        try {
-            $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
-        } catch (\RuntimeException $e) {
-            $this->topDir = '';
-        }
+        $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
```

Comment:
The problem is that `git rev-parse --show-toplevel` returns `128` as exit code when the command isn't executed inside a git directory, so it is transformed to an exception by `ProcessHelper::runCommand()`. I think you could play with its 2nd argument in order to see if it satisfies your expectations here.
```
$ cd /tmp
$ git rev-parse --show-toplevel
$ echo $?
```